common/src/KokkosKernels_Sorting.hpp

```diff
@@ -217,10 +193,11 @@ void bitonicSort(View v, const Comparator& comp) {
     Ordinal npot = 1;
     while (npot < n) npot <<= 1;
     // Partition the data equally among fixed number of teams
-    Ordinal chunkSize = 512;
-    Ordinal numTeams  = npot / chunkSize;
+    Ordinal chunkSize         = 512;
+    Ordinal numTeamsChunkSort = (n + chunkSize - 1) / chunkSize;
```

Comment:
This change is because the Kokkos sort_team exposed a bug where too many teams ran this functor, so chunks on some teams had negative length.